apps/public/src/app/(general)/bijlage/actions.ts

```diff
@@ -1,5 +1,86 @@
 'use server'
 
+import { getMeldingByMeldingIdAttachments, postMeldingByMeldingIdAttachment } from '@meldingen/api-client'
+import { cookies } from 'next/headers'
 import { redirect } from 'next/navigation'
 
-export const postAttachmentForm = async () => redirect('/contact')
+export const postAttachmentForm = async (fileList: FileList) => {
+  const cookieStore = await cookies()
+  const meldingId = cookieStore.get('id')
```

Comment:
Try to use the same patterns we use everywhere, or, when diverging, explain why and implement it everywhere. Different patterns for the same thing leads to hard to find bugs.